OJP/OJP_JourneySupport.xsd

```diff
@@ -89,7 +89,13 @@
 			</xs:element>
 			<xs:element name="TrainNumber" type="xs:normalizedString" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Contains the TrainNumber description from NeTEx. If several TrainNumber types exist, use the commercial number.</xs:documentation>
+					<xs:documentation>Contains the TrainNumber from NeTEx (TRAIN NUMBER from Transmodel). If several TrainNumber types exist, use the commercial one. In some cases also non-train modes
+will use TrainNumber.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element ref="siri:VehicleRef" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Contains the Vehicle reference of the vehicle. In Transmodel this may be VEHICLE.Id.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>Contains the Vehicle reference of the vehicle. In Transmodel this may be the VEHICLE Id.</xs:documentation>
```